_observability-plugin/ppl/commands.md

```diff
@@ -831,3 +831,84 @@ PPL query:
 ```ppl
 search source=my_index | where match(message, "this is a test", operator=and, zero_terms_query=all)
 ```
+
+## ad
+
+The `ad` command applies Random Cut Forest (RCF) algorithm in ml-commons plugin on the search result returned by a PPL command.Based on the input, two types of RCF algorithms will be utilized: fixed in time RCF for processing time-series data, batch RCF for processing non-time-series data.
```

Comment:
"The `ad` command applies **the** Random Cut Forest algorithm in **the** ML Commons plugin..."

Missing space after "by a PPL command." 

Also, since we want active voice, "Based on the input, the ML commons plugin uses two types of RCF algorithms...". And is this sentence missing an "and" to separate the two types of RCF algorithms?